tgui/packages/tgui/interfaces/ExaminePanel.js

```diff
@@ -3,6 +3,32 @@ import { Stack, Section, ByondUi } from '../components';
 import { Window } from '../layouts';
 import { resolveAsset } from '../assets';
 
+const formatURLs = (text) => {
+  const parts = [];
+  let regex = /https?:\/\/[^\s/$.?#].[^\s]*/gi;
+  let lastIndex = 0;
+
+  text.replace(regex, (url, index) => {
+    parts.push(text.substring(lastIndex, index));
+    parts.push(
+      <a
+        style={{
+          'color': '#0591e3',
```

Comment:
I'm curious, what's behind the choice of forcing the color to always be the same, for the URLs?

Reply:
The easiest way is like the decorations below.
You can make a separate style file, but in my opinion it’s too cool for that.